budgetportal/tests/test_data/test_management_commands_national_departments.csv

```diff
@@ -0,0 +1,41 @@
+government,department_name,vote_number,is_vote_primary,intro,website_url
```

Comment:
could you use more minimal test data so it's a bit easier to verify what's important? Doesn't have to be super realistic

Reply:
as discussed on Slack, will reduce the amount of data in the `intro` field